nixos.nix

```
@@ -115,6 +128,24 @@ in
             ''
               environment.persistence.files:
                   Currently, only files in /etc are supported.
+
+                  Please fix / remove the following paths:
```

```suggestion
                  Please fix or remove the following paths:
```

I feel like using `/` for "or" here is needlessly confusing (e.g. it may be mistaken for the root of a filesystem).

---

Ah, very true.